src/ert/config/gen_data_config.py

```diff
@@ -36,7 +35,9 @@ def from_config_list(cls, gen_data: List[str]) -> Self:
             )
 
         report_steps = rangestring_to_list(options.get("REPORT_STEPS", ""))
```

Comment:
If you define this as `report_steps: Optional[List[int]]` then you probably won't need the `type: ignore` on the next line.